roles/cam_disconnected/defaults/main.yml

```diff
@@ -0,0 +1,13 @@
+---
+cam_disconnected_location: "{{ workspace }}/restricted-network-operator-mirror"
+cam_disconnected_repo: "{{ lookup('env', 'CAM_DISCONNECTED_REPO') or '' }}"
+cam_disconnected_branch: "master"
+cam_disconnected_config_path: "{{ lookup('env', 'CAM_DISCONNECTED_CONFIG_PATH') | d(cam_disconnected_location + '/example-configs',true) }}"
+cam_disconnected_config: "{{ lookup('env', 'CAM_DISCONNECTED_CONFIG') or '' }}"
+rh_sub_user: "{{ lookup('env', 'SUB_USER') or '' }}"
+rh_sub_pass: "{{ lookup('env', 'SUB_PASS') | or '' }}"
```

Comment:
Is this `') | or ''` correct? Should the '|' be there?

Reply:
Yep resolved...